tools/run_tests/performance/scenario_result_schema.json

```diff
@@ -890,6 +890,31 @@
         "name": "core_http2_send_flowctl_per_write_99p", 
         "type": "FLOAT"
       }, 
+      {
```

Comment:
please note that adding more stats requires  running tools/run_tests/performance/patch_scenario_results_schema.py --bq_result_table on all datatests mentioned in https://github.com/grpc/grpc/blob/master/tools/internal_ci/linux/grpc_full_performance_release.sh and https://github.com/grpc/grpc/blob/master/tools/internal_ci/linux/grpc_full_performance_master.sh.
prior to submit.

(otherwise uploading of full benchmark results will get broken).